index.php

```diff
@@ -27,6 +27,46 @@
    exit;
 }
 
+//function to show warnings
+function warning($msg) { 
+	$script = "<script type='text/javascript'>$('#warning').fadeIn(1000);</script>";
+	$script .= "<script type='text/javascript'>$('#warning').html('<strong>Warning!</strong> $msg');</script>";
+	return $script;
+}
+
+//check if wol application (either wol or awake) is installed and set wol application to be used (e.g. for use with alpine & awake instead of wol)
+$wol_apps = [ "wakeonlan", "awake" ];//could be extended
+foreach ($wol_apps as $app) {
+  if (exec("command -v ".$app)<>"")
+  {
+    $wol = $app;//set the first application from the array that returns a non empty string as the wol application to be used
+    break;
+  }
+  if ($app === end($wol_apps))
+  {
+    $apps = implode(', ', $wol_apps);
+    $script = warning("unable to find any wol app, been looking for: <strong>".$apps."</strong> (one is necessary).<br>install one e.g. by running <strong>apt-get install wakeonlan</strong> or <strong>apk add awake</strong> (on the server)");//if non of the apps from the array are installed show a warning
+  }
+}
+
+//check if ping application is installed & working
+$ping_apps = [ "ping" ];//could be extended
+foreach ($ping_apps as $app) {
+  if (exec("command -v ".$app)<>"")
+  {
+    break;//just check it exists
+  }
+  if ($app === end($ping_apps))
+  {
+	$apps = implode(', ', $ping_apps);
+    $script = warning("Unable to find any ping app, been looking for: <strong>".$apps."</strong> (one is necessary).<br>Install one e.g. by running <strong>apt-get install iputils-ping</strong> or <strong>apk add iputils</strong> (on the server)");//if non of the apps from the array are installed show a warning
```

Comment:
Update this message similarly to above: `A suitable ping application ($apps) could not be found. Please install one.

As with above, the setup script should handle this (it already handles changing the ping permissions so that php can execute it).